ui/pages/Pool.tsx

```diff
@@ -0,0 +1,140 @@
+import * as addressStubs from 'stubs/address';
+import { POOL } from 'stubs/pools';
+import PoolInfo from 'ui/pool/PoolInfo';
+import isCustomAppError from 'ui/shared/AppError/isCustomAppError';
+import DataFetchAlert from 'ui/shared/DataFetchAlert';
+import AddressEntity from 'ui/shared/entities/address/AddressEntity';
+import * as PoolEntity from 'ui/shared/entities/pool/PoolEntity';
+import InfoButton from 'ui/shared/InfoButton';
+import LinkExternal from 'ui/shared/links/LinkExternal';
+import PageTitle from 'ui/shared/Page/PageTitle';
+import VerifyWith from 'ui/shared/VerifyWith';
+
+const Pool = () => {
+  const router = useRouter();
+  const appProps = useAppContext();
+  const hash = getQueryParamString(router.query.hash);
+
+  const { data, isPlaceholderData, isError, error } = useApiQuery('pool', {
+    pathParams: { hash, chainId: config.chain.id },
+    queryOptions: {
+      placeholderData: POOL,
+      refetchOnMount: false,
+    },
+  });
+
+  const addressQuery = useApiQuery('address', {
+    pathParams: { hash: data?.contract_address },
+    queryOptions: {
+      enabled: Boolean(data?.contract_address),
+      placeholderData: addressStubs.ADDRESS_INFO,
+    },
+  });
+
+  const content = (() => {
+    if (isError) {
+      if (isCustomAppError(error)) {
+        throwOnResourceLoadError({ resource: 'pool', error, isError: true });
+      }
+
+      return <DataFetchAlert/>;
+    }
+
+    if (!data) {
+      return null;
+    }
+
+    return (
+      <PoolInfo
+        data={ data }
+        isPlaceholderData={ isPlaceholderData }
+      />
+    );
+  })();
+
+  // More links can be added here
+  const externalLinks = [ {
+    url: data?.coin_gecko_terminal_url,
+    image: '/static/gecko_terminal.png',
+    title: 'GeckoTerminal',
+  } ].filter(link => Boolean(link.url));
+  const hasLinks = externalLinks.length > 0;
+
+  const externalLinksComponents = React.useMemo(() => {
+    return externalLinks
+      .map((link) => {
+        return (
+          <LinkExternal h="34px" key={ link.url } href={ link.url } alignItems="center" display="inline-flex" minW="120px">
+            <Image boxSize={ 5 } mr={ 2 } src={ link.image } alt={ `${ link.title } icon` }/>
+            { link.title }
+          </LinkExternal>
+        );
+      });
+  }, [ externalLinks ]);
+
+  const titleSecondRow = (
+    <Flex alignItems="center" justifyContent="space-between" w="100%">
+      { addressQuery.data ? <AddressEntity address={ addressQuery.data } isLoading={ addressQuery.isPlaceholderData }/> : <Box/> }
+      <Flex gap={ 2 }>
+        <InfoButton>
+          { `This Liquidity Provider (LP) token represents ${ data?.base_token_symbol }/${ data?.quote_token_symbol } pairing.` }
+        </InfoButton>
+        { hasLinks && (
+          <VerifyWith
+            links={ externalLinksComponents }
+            label=""
+            longText="View in"
+            shortText=""
+          />
```

Comment:
Seems like there is unnecessary top margin in the popover

<img width="328" alt="image" src="https://github.com/user-attachments/assets/da170501-e2a7-447c-8411-9a0bb4ac45d3" />